src/test/java/com/hocs/test/pages/homepage/Homepage.java

```diff
@@ -140,9 +190,149 @@ public void clickFirstInitialDraftAllocate() {
 
     // Multi Step Methods
 
-    // Assertions
-
+    public void firstStageFindMyCase() {
+        String thisCaseType =
+                Serenity.sessionVariableCalled("caseType").toString();
+        if (thisCaseType.equals("DCU MIN")) {
+            selectTeam1();
+            successfulCaseCreation.selectCaseReferenceNumberViaLinkText();
+        } else if (thisCaseType.equals("DCU TRO")) {
+            selectTeam1();
+            successfulCaseCreation.selectCaseReferenceNumberViaLinkTextAndStoreResultingElement();
+        } else if (thisCaseType.equals("DCU N10")) {
+            selectTeam3();
+            successfulCaseCreation.selectCaseReferenceNumberViaLinkTextAndStoreResultingElement();
+        } else {
+            System.out.println("The caseType is " + thisCaseType);
+        }
+    }
+
+    public WebElement currentCase() {
```

Comment:
this is its own method but its one line anyway, is there a reason for moving this to its own method ??

Reply:
Experiment gone wrong, removing.